.vscode/project-words.txt

```diff
@@ -44,6 +44,7 @@ nuage
 Nuwave
 OCIO
 Ojibway
+personalisation
```

Comment:
This is an accepted spelling on personalization and comes from `en-GB` which is preferred in GC Notify over the `en-US` spelling.